web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -50,102 +59,180 @@ const useStyles = makeStyles({
-                    return (
-                      <React.Fragment key={`fire-centre-${centreName}`}>
-                        <TableRow key={`fire-centre-${centreName}`}>
-                          <TableCell colSpan={3} className={classes.fireCentre}>
-                            {centre.name}
-                          </TableCell>
-                        </TableRow>
-                        {Object.entries(centre.planning_areas).map(([areaName, area]) => {
-                          return (
-                            <React.Fragment key={`zone-${areaName}`}>
-                              <TableRow
-                                className={classes.planningArea}
-                                key={`zone-${areaName}`}
-                              >
-                                <TableCell className={classes.planningArea} colSpan={3}>
-                                  {area.name}
-                                </TableCell>
-                              </TableRow>
-                              {Object.entries(area.stations).map(
-                                ([stationCode, station]) => {
-                                  return (
-                                    <TableRow
-                                      className={classes.station}
-                                      key={`station-${stationCode}`}
-                                    >
-                                      <TableCell key={`station-${station.code}-name`}>
-                                        {station.station_props.name} ({station.code})
-                                      </TableCell>
-                                      <TableCell
-                                        key={`station-${station.code}-elevation`}
-                                      >
-                                        {station.station_props.elevation}
-                                      </TableCell>
-                                      <TableCell
-                                        key={`station-${station.code}-fuel-type`}
-                                      >
-                                        {station.station_props.fuel_type.abbrev}
-                                      </TableCell>
-                                    </TableRow>
-                                  )
-                                }
-                              )}
-                            </React.Fragment>
-                          )
-                        })}
-                      </React.Fragment>
-                    )
-                  })}
-                </TableBody>
-              </Table>
-            </TableContainer>
-          </Paper>
-        </div>
-      )}
+      <div className={classes.controls}>
+        <Typography component="div" variant="subtitle2">
+          {props.title}
+        </Typography>
+        <Button color="primary" onClick={props.previousDay}>
+          Previous
+        </Button>
+        {props.currentDay}
```

Comment:
Looking at the wireframes - this is all falling away - otherwise I'd be asking we make this a nice date picker thing.

Reply:
What do you mean by falling away?

Reply:
There isn't going to be a "next"/"previous" button - the view is going to be weekly with tabs for each day (as per current design).

Reply:
Right ok, so no change until then?